src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/visitorders/VisitOrderBalanceMapping.kt

```diff
@@ -0,0 +1,48 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.visitorders
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.Transient
+import org.springframework.data.domain.Persistable
+import java.time.LocalDateTime
+
+class VisitOrderBalanceMapping(
```

Comment:
is it a visit order balance or just a visit balance?

Reply:
I can't decide.